flake.nix

```diff
@@ -608,23 +608,26 @@
                 docker volume prune -f
               '';
 
+            system-prune = pkgs.writeShellScriptBin "composable-system-prune" ''
+              ls ${docker-wipe-system}/bin/docker-wipe-system
+              rm /tmp/composable-finance/polkadot-launch/ -r -f && rm *.log -r -f
```

Comment:
```suggestion
              rm -rf /tmp/composable-finance/polkadot-launch  *.log
```

Reply:
obsolete, because will use zombienet and it generates randoms